#229 Add OpenTelemetry Collector Server Auth Extensions to Receivers (#2203)
grafana/alloy: docs/sources/reference/components/otelcol/otelcol.extension.jaeger_remote_sampling.md

  }
}
```

> ## Enabling Authentication

Reviewer: ```suggestion
## Enable authentication
```

> You can create a `jaeger_remote_sampling` extensions that requires authentication for requests. This is useful for limiting access to the sampling document. Note that not all OpenTelemetry Collector (otelcol) authentication plugins support receiver authentication. Please refer to the documentation for each `otelcol.auth.*` plugin to determine its compatibility.

Reviewer: ```suggestion
You can use `jaeger_remote_sampling` to authenticate requests.
This allows you to limit access to the sampling document.

 {{< admonition type="note" >}}
Not all OpenTelemetry Collector authentication plugins support receiver authentication.
Refer to the documentation for each `otelcol.auth.*` plugin to determine its compatibility.
 {{< /admonition >}}
```

The plugins referenced here... are the OTelCol-specific? Or do you mean Alloy `otelcol` components (eg `otelcol.auth.basic`, `otelcol.auth.bearer`, etc.)?

Author: I am referring to otelcol auth components here otelcol.auth.basic, etc. Should this be changed to not all otecol.auth.* components support receiver authentication.?

Reviewer: I think the question this raised for me was the term `plugin`. 

I've updated the suggested text above to reflect that it's the components we want to reference.  I also added a link back to

Author: Perfect. I will get this updated later today. You are correct in that in the context of Alloy they would be components, not plugins.

}
```

## Enable authentication

You can use `jaeger_remote_sampling` to authenticate requests.
This allows you to limit access to the sampling document.

>  {{< admonition type="note" >}}
> Not all OpenTelemetry Collector authentication plugins support receiver authentication.
> Refer to the [documentation](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/) for each `otelcol.auth.*` component to determine its compatibility.
>  {{< /admonition >}}

Reviewer: ```suggestion
{{< admonition type="note" >}}
Not all OpenTelemetry Collector authentication plugins support receiver authentication.
Refer to the [documentation](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/otelcol/) for each `otelcol.auth.*` component to determine its compatibility.
{{< /admonition >}}
```

Removing extra space
